src/views/ContinuationOut.vue

```diff
@@ -329,7 +389,7 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
   get showLoadingContainer (): boolean {
     // show loading container when data isn't yet loaded and when
     // no dialogs are displayed (otherwise dialogs may be hidden)
-    return (!this.dataLoaded && !this.saveErrorReason && !this.paymentErrorDialog)
+    return (!this.dataLoaded && !this.saveErrorReason)
   }
 
   /** Default comment (ie, the first line of the detail comment). */
```

Comment:
Line 403: schema says max length is 2000.

https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/continuation_out.json

Reply:
Oh shoot, great catch Sev. I just checked, the same applies for Consent. We might need to fix that there also.

Reply:
A two-fer!

Reply:
Thanks for fixing both. Please mention it in your ticket for Riyaz to have a look at.